### PR TITLE
Yuliia/wishlist page

### PR DESCRIPTION
**Pull request by ChebotarYuliia**

### CODE REVIEW

File: src/components/WishlistCard.jsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+              fontSize: { xs: '1rem', sm: '0.8rem', lg: '1rem' },
+            }}
+          >
+            {'Goal: ' + finalGoal + '₴'}
+          </Typography>
+          <Button
+            component={Link}
+            to={'/wishlist/' + id}
+            sx={{
+              fontWeight: 700,
```
Comment (ShevtsovVolodymyr) [inline src/components/WishlistCard.jsx:113]:
```suggestion
```

File: src/components/styled/ResponsiveContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  }
+  @media (min-width: 1200px) {
+    width: 1170px;
+  }
+  @media (min-width: 1536px) {
+    width: 1400px;
+  }
+  @media (max-width: 599px) {
+    width: 100%;
+  }
```
Comment (ShevtsovVolodymyr) [inline src/components/styled/ResponsiveContainer.jsx:26]: Even though its totally up to you, please consider using common breakpoints. Example:

320px — 480px: Mobile devices
481px — 768px: iPads, Tablets
769px — 1024px: Small screens, laptops
1025px — 1200px: Desktops, large screens
1201px and more —  Extra large screens, TV

or Bootstrap breakpoints:
 xs: 0,
  sm: 576px,
  md: 768px,
  lg: 992px,
  xl: 1200px,
  xxl: 1400px

**Review by ShevtsovVolodymyr — APPROVED** [review]

File: src/components/styled/ResponsiveContainer.jsx (hunk 2 above)
Comment (ChebotarYuliia) [reply src/components/styled/ResponsiveContainer.jsx:26]: all these breakpoints I took from MUI (https://mui.com/material-ui/customization/breakpoints/#main-content), except for 768px, it's needed just for more adaptability

File: src/components/WishlistCard.jsx (hunk 1 above)
Comment (ChebotarYuliia) [reply src/components/WishlistCard.jsx:113]: most of the design will be redone :)

File: src/components/styled/ResponsiveContainer.jsx (hunk 2 above)
Comment (ShevtsovVolodymyr) [reply src/components/styled/ResponsiveContainer.jsx:26]: Ok, great)

File: src/components/WishlistCard.jsx (hunk 1 above)
Comment (ShevtsovVolodymyr) [reply src/components/WishlistCard.jsx:113]: I mean prop is duplicated

File: src/redux/wishlist/wishlistActions.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from '../../services/axios/';
+
+export const getWishlistItems = createAsyncThunk(
```
Comment (AntonDeXy) [inline src/redux/wishlist/wishlistActions.js:4]: do we still need getWishlistItems when we have getSortedWishlistItems ?

Comment (ChebotarYuliia) [reply src/redux/wishlist/wishlistActions.js:4]: I don't think so)) will remove it

Comment (ChebotarYuliia) [reply src/redux/wishlist/wishlistActions.js]: done

File: src/redux/wishlist/wishlistActions.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import axios from '../../services/axios/';
+
+export const getSortedWishlistItems = createAsyncThunk(
+  '/wishlist-sorting',
+  async (arg, { rejectWithValue }) => {
+    const { page, field, order } = arg.options;
+    try {
+      const { data } = await axios.get(
+        `/api/wishlist/sorting/${page}/${field}/${order}`,
+      );
```
Comment (AntonDeXy) [inline src/redux/wishlist/wishlistActions.js:11]: as I asked u to use query params on backend, then it should be changed here too

**Pull request merged**